src/utils/blockchain.ts

```diff
@@ -134,6 +136,17 @@ const getRawTransaction = async (asset: string, txid: string) => {
     return await fetchBlockExplorer<string>(asset, `/tx/${txid}/hex`);
 };
 
+export const getBlockTipHeight = async (asset: string) => {
+    return await fetchBlockExplorer<string>(asset, "/blocks/tip/height");
+};
+
+export const getFeeEstimationsExternal = async (asset: string) => {
+    return await fetchBlockExplorer<Record<string, number>>(
+        asset,
+        "/fee-estimates",
+    );
+};
```

Comment:
This does not exist on the mempool.space API
<img width="314" height="49" alt="image" src="https://github.com/user-attachments/assets/1773f4fa-00d6-46bb-8d9d-4adedf71b1c3" />

Which would leave blockstream.info as single point of failure here

Also, the esplora fee estimations are super weird, on fee spikes. How about trying or API first, and then falling back to mempool.space and if that doesn't work either, trying Esplora?